confignode/src/main/java/org/apache/iotdb/confignode/procedure/env/ConfigNodeProcedureEnv.java

```diff
@@ -203,9 +203,6 @@ public void removeConfigNodePeer(TConfigNodeLocation tConfigNodeLocation)
     try {
       // Execute removePeer
       if (getConsensusManager().removeConfigNodePeer(tConfigNodeLocation)) {
-        configManager
-            .getLoadManager()
-            .removeNodeHeartbeatHandCache(tConfigNodeLocation.getConfigNodeId());
```

Comment:
Why remove this

Reply:
I suppose that there is not need to make the cache management more intricate. Because the NodeCache and RegionGroupCache are limited and they will be discarded as long as the ConfigNode-leader changed.